src/main/java/de/cyface/collector/model/Measurement.java

```diff
@@ -1,305 +0,0 @@
-     */
-    public File getBinary() {
-        return binary;
-    }
-
-    /**
-     * @return A JSON representation of this measurement.
-     */
-    public JsonObject toJson() {
-        final var ret = new JsonObject();
-
-        ret.put(FormAttributes.DEVICE_ID.getValue(), metaData.getDeviceIdentifier());
-        ret.put(FormAttributes.MEASUREMENT_ID.getValue(), metaData.getMeasurementIdentifier());
-        ret.put(FormAttributes.OS_VERSION.getValue(), metaData.getOperatingSystemVersion());
-        ret.put(FormAttributes.DEVICE_TYPE.getValue(), metaData.getDeviceType());
-        ret.put(FormAttributes.APPLICATION_VERSION.getValue(), metaData.getApplicationVersion());
-        ret.put(FormAttributes.LENGTH.getValue(), metaData.getLength());
-        ret.put(FormAttributes.LOCATION_COUNT.getValue(), metaData.getLocationCount());
-        if (metaData.getLocationCount() > 0) {
-            ret.put("start", geoJson(metaData.getStartLocation()));
-            ret.put("end", geoJson(metaData.getEndLocation()));
-        }
-        ret.put(FormAttributes.MODALITY.getValue(), metaData.getModality());
-        // We can only store the usedId as string as:
-        // - `new ObjectId(userId)` inserts `{timestamp:1654072354, date:1654072354000}` into the database
-        // - `new JsonObject().put("$oid", userId))` leads to an exception: Invalid BSON field name $oid
-        ret.put(USER_ID_FIELD, userId);
-        ret.put(FormAttributes.FORMAT_VERSION.getValue(), metaData.getFormatVersion());
-
-        return ret;
-    }
-
-    /**
-     * Converts a location record into {@code JSON} which supports the mongoDB {@code GeoJSON} format:
-     * https://docs.mongodb.com/manual/geospatial-queries/
-     *
-     * @param record the location record to be converted
-     * @return the converted location record as JSON
-     */
-    private JsonObject geoJson(final RequestMetaData.GeoLocation record) {
-        final var lat = record.getLatitude();
-        final var lon = record.getLongitude();
-        final var ts = record.getTimestamp();
-        final var geometry = new JsonObject()
-                .put("type", "Point")
-                .put("coordinates", new JsonArray().add(lon).add(lat));
-        return new JsonObject()
-                .put("location", geometry)
-                .put("timestamp", ts);
-    }
-
-    /**
-     * @return A codec encoding and decoding this <code>Measurement</code> for usage on the event bus.
-     */
-    public static MessageCodec<Measurement, Measurement> getCodec() {
-        return new EventBusCodec();
-
-    }
-
-    /**
-     * A <code>MessageCodec</code> implementation to be used for transmitting a <code>Measurement</code> via a Vertx
-     * event bus.
-     *
-     * @author Klemens Muthmann
-     * @author Armin Schnabel
-     * @version 2.0.0
-     * @since 1.0.0
-     */
-    static final class EventBusCodec implements MessageCodec<Measurement, Measurement> {
```

Comment:
I guess we don't need this anymore, or did this move somewhere I did not find it?

It's not in the `Measurment.kt` anymore at the end of the PR.